src/styles/variables/variables.scss

```
@@ -0,0 +1,3 @@
+$main-color: #060b35;
+$second-color: #616070;
+$blue-accent-color: #00acdc;
```

It is better to use more universal naming.
```suggestion
$accent-color: #00acdc;
```